Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -965,6 +1151,17 @@ public SessionDetails MakeSessionHandleFromSearch(string sessionId)
 
         #region Online Session Lookup and Search
 
+        /// <summary>
+        /// Performs an online Search to look up a Session, where all attributes provided are met.
+        /// <see cref="OnFindSessionsCompleteCallback"/> is called with the results.
+        /// A handle for retriving results will be stored in <see cref="CurrentSearch"/>.
+        /// 
+        /// TODO: I feel like this should run a callback or return a Result in case of errors when trying to set up the search.
+        /// TODO: Only finds up to 10 results. Should have some method for finding next sets of results.
```

Comment:
This is a great example of what I think of as an appropriate `TODO:` comment. It clearly states what needs to be changed, and gives context for why.